### ISSUE TITLE
Doesn't work in MS Edge or IE11

### ISSUE THREAD

**Issue by careyestes** [ETCDEVTeam/hashicon#28]
Load up https://etcdevteam.github.io/hashicon/examples/ in Edge or IE and you will see it doesnt load the icons. Ive implemented it using React and get a few js errors in the console. 

Will support for Edge and IE be added?

**Comment by oori** [comment]
Yes.  The current code is written in modern javascript (ES6), we need to transpile for older browsers (*I'm surprised IE Edge doesn't handle it..*).     
TODO: Babel (as rollup plugin) will solve this.

**Comment by cwbutler** [comment]
@oori, I see that babel is commented out in the rollup config file. Is this intentional? Will uncommenting the plugin fix?

**Comment by oori** [comment]
Nope..  Using "rollup-plugin-babel" is the start, but requires more.
Lack of time and need pushes this down on my TODO list, but I'll try to do it soon.
PR welcome.
